mackup/applications/popcorn-time.cfg

```diff
@@ -0,0 +1,9 @@
+[application]
+name = Popcorn-Time
```

Comment:
![screenshot 2015-02-27 19 09 36](https://cloud.githubusercontent.com/assets/68406/6417171/40584fa0-beb4-11e4-8d60-e2011bd48a34.png)
.app is with dash but I rather prefer yours as well.